wiki/Community/Mappers_Guild/es.md

```diff
@@ -0,0 +1,74 @@
+---
+tags:
+  - Mapper's Guild
+  - MG
+---
+
+# Mappers' Guild
+
+![](img/mg-logo.png)
+

```

Comment:
```suggestion
---
tags:
  - Mapper's Guild
  - MG
---

# Mappers' Guild

![](img/mg-logo.png)

```

Reply:
Is there any way you could DM me this changes? Github doesn't let me apply the changes since there are no actual changes to the lines themselves